fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -980,19 +1172,19 @@ static path_redirect_info ShouldRedirectImpl(const CharT* path, redirect_flags f
 	// Basically, what goes into RedirectedPath here also needs to go into 
 	// FindFirstFileFixup.cpp
     auto vfspath = NormalizePath(path);
-    vfspath = VirtualizePath(std::move(vfspath));
+    vfspath = VirtualizePath(std::move(vfspath),inst);
     if (vfspath.drive_absolute_path != NULL)
     {
-        LogString(L"\t\tFRF Virtualized", vfspath.drive_absolute_path);
+        LogString(inst, L"\t\tFRF Virtualized", vfspath.drive_absolute_path);
     }
 
     // Figure out if this is something we need to redirect
     for (auto& redirectSpec : g_redirectionSpecs)
     {
-        LogString(L"\t\tFRF Check against: base", redirectSpec.base_path.c_str());
+        //LogString(inst, L"\t\tFRF Check against: base", redirectSpec.base_path.c_str());
         if (path_relative_to(vfspath.drive_absolute_path, redirectSpec.base_path))

```

Comment:
Just as a thought, but I don't like this giant nested if-else blocks.  I don't know how to get rid of it at the moment, but I would like them taken away in the future.

Reply:
Not saying the if-else chain is anyones fault.  Just voicing my thoughts.

Reply:
Yeah, I hear you. I think this is still short enough to be managable without breaking out more functions.

Reply:
Coolio.